Script to download wasm from CI

dskloetd opened the pull request:
# Motivation

We want to make the release a less manual process.
One of the steps in the release process is to download the WASM from CI.

# Changes

Add a script to download the nns-dapp WASM from CI.

# Tests

Ran the script, also with wrong commit and wrong filename to exercise the error code paths.

Added a test which checks that the downloaded file is a gzip file.

bitdivine commented:
Looks good, but can you include a test, please?  E.g. pull head of main.  Doesn't need to be in this PR.

It looks as if you plan to use this as part of the release process.  The release process for nns-dapp can also be run against a testnet or local.

bitdivine reviewed (approved):
Looks good, but can you include a test, please?  E.g. pull head of main.

It looks as if you plan to use this as part of the release process.  The release process for nns-dapp can also be run against

dskloetd commented:
I added `scripts/nns-dapp/download-ci-wasm.test` which tests that the downloaded file is a gzip file and added it to `.github/workflows/checks.yml`.
Please have another look.
